lib/theme_check/language_server/completion_providers/object_completion_provider.rb

```diff
@@ -3,29 +3,32 @@
 module ThemeCheck
   module LanguageServer
     class ObjectCompletionProvider < CompletionProvider
-      def completions(content, cursor)
-        return [] unless (variable_lookup = variable_lookup_at_cursor(content, cursor))
+      def completions(relative_path, line, col)
```

Comment:
I'm not sure how exact the completion provider needs to be (after all, VS Code already displays the error _Expected end_of_string but found dotdot_), so is it okay if completions are also suggested when syntax is erroneous (e.g. `dotdot`)?

![14-57-qxm05-lu0l0](https://user-images.githubusercontent.com/12963280/201617895-a27ec1ea-5725-4606-867a-28bdd6c608fe.jpg)